demos/twa-offline-first/build.gradle

```diff
@@ -0,0 +1,50 @@
+/*
+ *    Copyright 2019 Google LLC
```

Comment:
Nit: could you go through all of the copyright headers and update the year to 2021 please?